src/main/groovy/de/dkfz/roddy/execution/jobs/SubmissionCommand.groovy

```diff
@@ -38,6 +38,7 @@ abstract class SubmissionCommand extends Command {
 
         // collect parameters for job submission
         List<String> parameters = []
+        parameters << getEnvironmentExportString()
```

Comment:
Look at this again.

Reply:
Maybe make this optional?

Reply:
I can create another "request" or "enable" flag. Then it'd be configurable.

Reply:
I think this would be good. I just had a case, where my tmux session inherited the modules config from before the tmux call. Then I changed the modules, which was supposed to activate some tools, but didn't, because implicitly (invisibly to me at first) the module was loaded already. This eventually caused the same problem on the execution host, because upon submission, my local problem was exported there. So yes, make it possible to enable/disable this.

What should be the default? I guess "off" for BE. And for Roddy?